client/signup/steps/domains/index.jsx

```diff
@@ -713,10 +711,6 @@ export default connect(
 
 		return {
 			designType: getDesignType( state ),
-			// no user = DOMAINS_WITH_PLANS_ONLY
-			domainsWithPlansOnly: getCurrentUser( state )
```

Comment:
I removed this since it's already being passed from the parent component, but I'd appreciate a double check.